README.rst

```diff
@@ -41,49 +41,150 @@
 MAPIE - Model Agnostic Prediction Interval Estimator
 ====================================================
 
-**MAPIE** allows you to easily estimate prediction intervals (or prediction sets)
-using your favourite scikit-learn-compatible model for
-single-output regression or multi-class classification settings.
+Quantifying the uncertainties and controlling the risks of ML model predictions is of crucial importance
+for developing and deploying reliable artificial intelligence (AI) systems. Uncertainty quantification (UQ)
+involves all the stakeholders who develop and use AI models.
 
-Prediction intervals output by **MAPIE** encompass both aleatoric and epistemic
-uncertainties and are backed by strong theoretical guarantees thanks to conformal
-prediction methods [1-7].
+**MAPIE** is an open-source Python library hosted on scikit-learn-contrib project that allows you to:
 
+- easily **estimate conformal prediction intervals** (or prediction sets) given a degree of confidence or risk
+  for single-output regression, binary and multi-class classification settings [3-9].
+- easily **control risks** (such as coverage, recall or any other non-monotone risk) by estimating
+  relevant prediction sets for multi-label classification and beyond [10-12].
+- easily **wrap your favorite scikit-learn-compatible model** for the purposes just mentioned.
 
-🔗 Requirements
-===============
+Here's a quick instantiation of MAPIE models for regression and classification problems related to uncertainty quantification
+(more details in the Quickstart section):
+
+.. code:: python
+
+    # Uncertainty quantification for regression problem
+    from mapie.regression import MapieRegressor
+    mapie_regressor = MapieRegressor(estimator=regressor, method='plus', cv=5)
+
+.. code:: python
+
+    # Uncertainty quantification for classification problem
+    from mapie.classification import MapieClassifier
+    mapie_classifier = MapieClassifier(estimator=classifier, method='score', cv=5)
+
+.. code:: python
```

Comment:
I would add multi-label. I think let's keep it to two simple use-cases. Regression and classification?